src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/appsettings.json

```diff
@@ -1,14 +1,14 @@
 {
   "GeneralSettings": {
-    "HostName":"at21.altinn.cloud",
+    "HostName": "at21.altinn.cloud",
     "SBLCookieName": ".ASPXAUTH",
     "BaseUrl": "https://at21.altinn.cloud",
     "BridgeApiEndpoint": "https://at21.altinn.cloud/sblbridge/authentication/api/",
     "SBLRedirectEndpoint": "https://at21.altinn.cloud/ui/authentication",
     "PlatformEndpoint": "http://localhost:5040/",
     "ClaimsIdentity": "UserLogin",
-    "JwtCookieValidityTime": "200",
-    "AltinnPartyCookieName":"AltinnPartyId"
+    "JwtCookieValidityTime": "30",
```

Comment:
Can we use config here?

Reply:
@TheTechArch i don't get you, appsettings.json is a config right

Reply:
I dont get my self.